apps/chromium-browser/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  chromium:
+    image: lscr.io/linuxserver/chromium:stable
+    container_name: chromium
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Etc/UTC
+      - CHROME_CLI=https://www.linuxserver.io/ #optional
```

Comment:
This line can be removed as it is just the default website to be opened, when chrome starts
```suggestion
```